### PR TITLE
Reformat code

### PR DESCRIPTION
**Pull request by rey**
This is a first pass of cleaning up the `scss` files in terms of formatting. I appreciate this is somewhat opinionated but has been done to standardise all the files in the repo and to improve readability :pray:
- Indents use (2) spaces only (previously there was a mixture of spaces and tabs)
- Strip trailing white space for sanity
- One line per declaration for readability
- Use `//` comment syntax instead of `/* */` for readability

### CODE REVIEW

**Comment by sdougbrown** [comment]
I prefer tabs or 4 spaces, but this is all the cleanup that I meant to do a while ago - thanks.   I'll go through and double-check a bit later today. :smile:

**Comment by rey** [comment]
@sdougbrown :thumbsup:

File: _buttons.scss [hunk 1]
```diff
@@ -1,152 +1,183 @@
...
+}
+.button:active,
+.butt:active,
+.btn:active,
+%btn:active {
+  background-color: #eee;
+}
+
+// if you click me, do I not bleed?
+.button:active, .button:focus {
```
Comment (sdougbrown) [inline _buttons.scss]: gotcha - split to two lines :)

File: _text.scss [hunk 2]
```diff
@@ -1,176 +1,370 @@
...
-.t-xxs                  { font-size: $t-2xs; }
-.t-xxxs                 { font-size: $t-3xs; }
-.t-xxxxs                { font-size: $t-4xs; }
-.t-xxxxxs, .t-half      { font-size: $t-5xs; }
-.t-n                    { font-size: $t-n; }
 
-/* careful with this... */
-.t-nat { font-size: 1rem; }
+// Element styling
+h1, h2, h3, h4, h5, h6, p, ul, ol, blockquote { padding-top: 0.5em; padding-bottom: 0.5em; }
```
Comment (sdougbrown) [inline _text.scss]: this should probably be split into multiple lines as well

File: _text.scss [hunk 3]
```diff
@@ -1,176 +1,370 @@
...
-	.t-n--screen-m			{ font-size: 1em; }
-    
-    .t-mid--screen-m, .t-center--screen-m { text-align: center; }
-    .t-left--screen-m { text-align: left; }
-    .t-right--screen-m { text-align: right; }
+@media only screen and (max-width: $media--m) {
+  .h-xxl--screen-m,
+  .t-xxxxl--screen-m,
+  .t-triple--screen-m {
+    font-size: 3em;
```
Comment (sdougbrown) [inline _text.scss]: not your fault, but I just noticed in going through that we moved all of these sizes to vars, but not in the media queries.  oops.  we should probably update that!

**Comment by sdougbrown** [comment]
Yeah I'm happy with the cleanup in general, but I don't know if I can live with the 2 spaces thing. :smile:   I personally prefer tabs in my CSS files, but there's some internal resistance to that and I feel like I'm losing the battle there.  Could we go to 4 spaces instead?

Just a heads-up, #2 is going to result in a few conflicts simply because some things are moving around - they should be easy to resolve.

**Comment by rey** [comment]
> Yeah I'm happy with the cleanup in general, but I don't know if I can live with the 2 spaces thing.

I work with the `.sass` syntax (spits an error if you _don't_ use 2 spaces). That said, I'm quite flexible -- is there a common consensus amongst the @duckduckgo repos with regards to tabs/spaces/etc? It would be nice to be inline with other repos in the org.

> Just a heads-up, #2 is going to result in a few conflicts simply because some things are moving around - they should be easy to resolve.

I spotted that, shouldn't be too much of a head scratcher :smile:

File: _text.scss (hunk 2 above)
Comment (rey) [inline _text.scss]: @sdougbrown d2edb746c215d03d2cc0256a7c30552174ac86b6 :+1:

**Comment by sdougbrown** [comment]
>  is there a common consensus amongst the @duckduckgo repos with regards to tabs/spaces/etc?

Yeah I went looking and it turns out we have a code styleguide!  https://duck.co/duckduckhack/code_styleguide

It says that 4 spaces is the standard in _DuckDuckHack_ repos, and this is technically not a DuckDuckHack (i.e. Instant Answer) project so we're free to set our own standards but it'd probably be easier to go along with this. :smile:

**Comment by sdougbrown** [comment]
I've created more conflicts for you in #5 based on the default vars conversation we had. :smile:

**Comment by rey** [comment]
@sdougbrown
- Previously enforced a "one declaration per line" rule but since realised that this doesn't work on files like `_text.scss` and `_litestrap.scss` to the point where it harms readability. I believe this should be revisited at a later date -- for consistency right now, I've made all declarations inline across the repo.
- Change `_vars.scss` to `_variables.scss` for better readabilty.
- Change 2 spaces to 4 spaces (standard in DuckDuckHack repos).

**Comment by sdougbrown** [comment]
It looks like you just added `_variables` instead of renaming the file.  I've done that before, just a git error.

**Comment by rey** [comment]
@sdougbrown :+1: https://github.com/rey/litestrap/commit/089ddc7d207ae647bea6c33ea103d8e87f3c2fd0

**Comment by sdougbrown** [comment]
Thank you for the time you spent reformatting all of this.  I don't see any further issues, and I'm happy with where it's at.  :+1:

**Pull request merged**